Purchases/Public/Purchases.swift

```diff
@@ -1008,7 +1008,8 @@ public extension Purchases {
      * - Parameter apiKey: The API Key generated for your app from https://app.revenuecat.com/
      *
      * - Parameter appUserID: The unique app user id for this user. This user id will allow users to share their
-     * purchases and subscriptions across devices. Pass nil if you want `Purchases` to generate this for you.
+     * purchases and subscriptions across devices. Pass `nil` if you want `Purchases` to generate this for you.
+     * An empty `appUserID` is equivalent to `nil`.
```

Comment:
nitpick: perhaps this would be clearer as "Pass `nil` or an empty string if ..."

Reply:
[not for this PR] Just realized that this is yet another set of instances where we might wanna update the `Purchases` references to `RevenueCat`

Reply:
You mean the whole `Purchases` class? Yeah.

Reply:
> nitpick: perhaps this would be clearer as "Pass nil or an empty string if ..."
Oh yeah much simpler. changed.